po/de.po

```diff
@@ -3199,11 +3264,10 @@ msgstr ""
 "  * Akzeptiert immer eine feste Anzahl von Parametern.\n"
 "  * Akzeptiert immer einen einzelnen Satz von Parametertypen.\n"
 "* Standardwerte werden nicht unterstützt:\n"
-"  * Alle Aufrufseiten haben die gleiche Anzahl von Argumenten.\n"
+"  * Alle Aufrufseiten (call sites) haben die gleiche Anzahl von Argumenten.\n"
```

Comment:
```suggestion
"  * Alle Aufrufstellen (call sites) haben die gleiche Anzahl von Argumenten.\n"
```

Reply:
Just a tip: you can use "Add suggestion to batch" on your own suggestions and commit them directly. I often do this for typo fixes where I'm pretty sure that the PR author won't be upset with my change :smile:  After committing my own suggestion like this, I then go ahead and merge the PR. That saves a few round trips for everybody.